scripts/rollup/plugins.mjs

```diff
@@ -91,8 +91,11 @@ export const makeOutputPlugins = ({ isProduction, extension }) => {
     throw new Error('Missing option `extension` on makeOutputPlugins({ ... })');
 
   return [
-    isProduction && replace({
-      'process.env.NODE_ENV': JSON.stringify('production')
+    replace(isProduction ? {
+      'process.env.NODE_ENV': JSON.stringify('production'),
+      '__pkg_version__': JSON.stringify(settings.pkg.version),
```

Comment:
I lost track of this, but not 100% sure where this would be used in the devtools :thinking: